app/boards/shields/launchpad/launchpad.conf

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+CONFIG_BT_MAX_CONN=6
```

Comment:
It looks like the default keymap doesn't have options for using six bluetooth connections. Unless there's a good reason to do otherwise, I think we should leave this at the default.